metrics-facade-base/src/main/java/com/ringcentral/platform/metrics/producers/AbstractMetricsProducer.java

```diff
@@ -41,22 +44,62 @@ protected MetricName nameWithSuffix(String... suffix) {
         return namePrefix.isEmpty() ? MetricName.of(suffix) : MetricName.of(namePrefix, suffix);
     }
 
+    protected Supplier<ObjectVarConfigBuilder> objectVarConfigBuilderSupplier(
+            String description, MetricDimension... dimensions
+    ) {
+        final var builder = objectVarConfigBuilder();
+        setDescriptionAndDimensions(builder, description, dimensions);
+        return () -> modified(builder);
+    }
+
     protected Supplier<ObjectVarConfigBuilder> objectVarConfigBuilderSupplier() {
```

Comment:
This method is not used anymore. What about removing it? Same for the other xVarConfigBuilderSupplier() methods.

Reply:
Removed, thank you.